src/replica/replica.h

```diff
@@ -78,6 +78,14 @@ namespace test {
 class test_checker;
 }
 
+enum manual_compaction_status
+{
+    CompactionFinish = 0,
```

Comment:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
> https://google.github.io/styleguide/cppguide.html#Enumerator_Names

It says "Enumerators (for both scoped and unscoped enums) should be named like constants, not like macros. That is, use kEnumName not ENUM_NAME." :(

Reply:
That is my mistake, now it is fixed.